clients/src/main/java/org/apache/kafka/common/serialization/UUIDDeserializer.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.common.serialization;
+
+import org.apache.kafka.common.errors.SerializationException;
+
+import java.io.UnsupportedEncodingException;
+import java.util.Map;
+import java.util.UUID;
+
+public class UUIDDeserializer implements Deserializer<UUID> {
+    private String encoding = "UTF8";
```

Comment:
I would expect this to be `UTF-8` with a dash. That's the format in https://docs.oracle.com/javase/7/docs/api/java/nio/charset/Charset.html

Reply:
@jklukas interesting, i assumed this was correct because this is what [StringSerializer](https://github.com/brandonkirchner/kafka/blob/02cd4feed2f6857ba095f56160d1c6a9f56ca2ea/clients/src/main/java/org/apache/kafka/common/serialization/StringSerializer.java#L29) uses. Should i correct those as well?